doc/functions.sgm

```diff
@@ -532,7 +532,31 @@
             </example>
 
           </sect3>
-            
+          <sect3 id="funcs.spoly.isconvex">
+            <title>
+              Spoly is convex
+            </title>
+            <para>
+              Returns true if the specified spherical polygon is convex. Returns false otherwise.
```

Comment:
Is this line longer than 79 characters? I can't tell looking at it in the web browser here. If it is, please word-wrap it.